src/lib/interface/robot_interface.py

```diff
@@ -0,0 +1,74 @@
+"""
+This module just contains the RobotInterface class. It provides utility functions to interact with
+motors on the robot. 
+"""
+
+from rclpy.node import Node
+from rclpy.publisher import Publisher
+from std_msgs.msg import String
+
+from lib.configs import MoteusMotorConfig, MotorConfigs
+from lib.moteus_motor_state import MoteusRunSettings
+
+
+class RobotInterface:
+    """
+    A class that provides utility functions to interact with motors on the robot
+    """
+
+    def __init__(self, ros_node: Node) -> None:
+        self._ros_node = ros_node
+        self._publishers: dict[int, Publisher] = {}
+
+        for motor_config in MotorConfigs.getAllMotors():
+            self._ros_node.create_publisher(String, motor_config.getInterfaceTopicName(), 10)
+
+    def runMotor(self, motor: MoteusMotorConfig, run_settings: MoteusRunSettings) -> None:
```

Comment:
Do we leave the MoteusRunSettings in terms of moteus units, or do we want to have the MoteusRunSettings to be in terms of "engineering" units? Something to discuss